spec/system/welcome_spec.rb

```diff
@@ -2,19 +2,13 @@
 
 require "rails_helper"
 
-RSpec.describe "WelcomeController" do
+RSpec.describe "WelcomeController", stub_mediaflux: true do
   context "unauthenticated user" do
     it "shows the 'Log In' button" do
       visit "/"
       expect(page).to have_content "Welcome to the TigerData user portal"
       expect(page).to have_content "Log In"
     end
-
-    it "supports JS" do
```

Comment:
This was testing core Rails functionality, and the button was removed with the merging of https://github.com/pulibrary/tiger-data-app/pull/114 (hence, it would continue to fail).